7.1/contrib/etc/scl_enable

```diff
@@ -3,4 +3,4 @@
 #
 # This will make scl collection binaries work out of box.
 unset BASH_ENV PROMPT_COMMAND ENV
-source scl_source enable nginx php-fpm
+source scl_source enable nginx php-fpm 2>&1 >> /dev/null
```

Comment:
I believe that I had to do this to stop a harmless warning message coming out of this command about "cannot see that nginx is configured" which would go to stdout when save-artifacts ran. save-artifacts then tars a file to stndout. the next build attempts to read a tar from stdin. That crashed due to the warning message. Took a while and a stackoverflow answer from a expert on s2i to fix that.